docs/embedded/concepts/app-concepts/containertypes.md

```diff
@@ -22,17 +22,34 @@ The permission mapping between a ContainerTypeID-AppID pair determines the opera
 > [!TIP]
 > To learn more about application architecture, see [Application Architecture](./app-architecture.md).
 
-## SharePoint Embedded Trial Container Types
+
+## TODO: Introduce two types
+* introduce the limits and difference between trial and billable container type
+* each type has different requirements
+
+### Create a SharePoint Embedded Paid ContainerType
+links: register-ct tutorial
+
+
+
+### SharePoint Embedded Trial Container Types
 
 To aid customers with the development of applications to be used with SharePoint Embedded, we're allowing the creation of a Trial Container Type. Customers are allowed one Trial Container Type at any time in their tenant. The Trial Container Type won't be linked to billing and can only be consumed by the Partner tenant.
 
 > [!NOTE]
 > The Trial Container Type is the only Container Type that is restricted to the Partner (creating) tenant and cannot be shared with other consuming tenants.
 
+### Create a Trial Container Type: Visual Studio Code Extension
+You can create a Trial Container Type using the SharePoint Embedded Visual Studio Code Extension [here](https://marketplace.visualstudio.com/items?itemName=SharepointEmbedded.ms-sharepoint-embedded-vscode-extension)
```

Comment:
MS Docs recommend linking the text of the target, not words like 'click this link' or 'here'.